.github/workflows/test-e2e.yml

```diff
@@ -0,0 +1,31 @@
+name: Tests E2E
+#  This workflow is run on pushes to main & every Pull Requests where a .go, .mod, .sum have been changed
+on:
+  pull_request:
+    paths:
+      - "**/*.go"
+      - "go.mod"
+      - "go.sum"
+  push:
+    branches:
+      - main
+    paths:
+      - "**/*.go"
+      - "go.mod"
+      - "go.sum"
+
+permissions:
+  contents: read
+
+jobs:
+  test-e2e:
+    runs-on: ubuntu-latest
+    needs: build
```

Comment:
this can call cross workflow?

Reply:
Nope, my bad, this is a typo. We don't require build `go test` anyway.